index.html

```diff
@@ -4,9 +4,146 @@
+                <li><a href="#winners" class="sidebarNavigation">Last Winners</a></li>
+                <li><a href="#sponsors" class="sidebarNavigation">Our Sponsors</a></li>
+            </ul>
+        </div>
+    </div>
+
+  </header>
+  <main>
+    <section id="headline" class="headline">
+      <div class="headline-title">
+        <p>"Hello! Valorant Players"</p>
+        <h1>VCT VALORANT MASTERS TOURNAMENT 2021</h1>
+      </div>
+      <div class="headline-description">
+        <p>Let's start with Challengers. Regions around the world will compete in Challengers events in three stages throughout the year. Each stage features two Challengers main events, which any team from the region can qualify for through open qualifiers. At the main events, teams compete for a place in the Challenger qualifiers. Teams that make it to and win the Challengers qualifiers will qualify for a global competition at the Masters events.</p>
+        <h2>2021.07.15(THU) ~ 16(MON)</h2>
+        <p class="address">@ Reykjavík, Iceland, Venue: Laugardalshöll</p>
+      </div>
+    </section>
+
+    <section id="classify" class="classify-steps">
+      <h1>Step to Classify to Champions</h1>
+      <div class="divider"></div>
+      <ul class="classify-steps-cards">
+        <li>
+          <img src="./assets/icon/register.svg" alt="">
+          <h2>Register your team</h2>
+          <p>You have to send the details of your team by email</p>
+        </li>
+        
+        <li>
+          <img src="./assets/icon/group.svg" alt="">
+          <h2>Group Phase</h2>
+          <p>All the teams register start to fight each others to classify</p>
+        </li>
+        
+        <li>
+          <img src="./assets/icon/remove.svg" alt="">
+          <h2>Eliminatories</h2>
+          <p>All the teams that classify start to fight in the eliminatories</p>
+        </li>
+        
+        <li>
+          <img src="./assets/icon/winner.svg" alt="">
+          <h2>Win the Master</h2>
+          <p>Only 1 team win the master</p>
+        </li>
+        
+        <li>
+          <img src="./assets/icon/valorant-logo.svg" alt="">
+          <h2>Champions Start</h2>
+          <p>Adter 3 master the 3 champions fight eachothers in the CHAMPIONS</p>
+        </li>
+      </ul>
+      <a href="about.html">More Info </a>
+    </section>
+
+    <section id="winners" class="winners">
+      
+    </section>
+
+    <section id="sponsors" class="sponsors">
+      <h1>Sponsor By</h1>
+      <div class="divider"></div>
+      <ul>
+        <li>
+          <img src="./assets/icon/riot-games-logo.svg" alt="">
+        </li>
+        <li>
+          <img src="./assets/icon/cisco.png" alt="">
+        </li>
+        <li>
+          <img src="./assets/icon/coca-cola.png" alt="">
+        </li>
+        <li>
+          <img src="./assets/icon/lg.png" alt="">
+        </li>
+      </ul>
+    </section>
+  </main>
+  <footer>
```

Comment:
- [ ] **The overall layout of the footer does not match the original design. Please fix it.**

Original Design|Cloned Design
:--------------------:|:-----------------:
![image](https://user-images.githubusercontent.com/59168430/124976432-e5a71200-e04c-11eb-934f-345564c00a2c.png)|![image](https://user-images.githubusercontent.com/59168430/124976505-fc4d6900-e04c-11eb-9cbb-bd21122c2def.png)

The original design contains a logo aligned to the left and text aligned to the right. Please modify your design to match the original.